modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift-sdn/configuring-egress-firewall.adoc
+
+[id="nw-egressnetworkpolicy-about_{context}"]
+= How an egress firewall works in a project
+
+As an {product-title} cluster administrator, you can use an _egress firewall_ to
+limit the external IP addresses that some or all Pods can access from within the
+cluster. Egress firewall policy supports the following scenarios:
+
+- A Pod can only connect to internal hosts, and cannot initiate connections to
+the public Internet.
+- A Pod can only connect to the public Internet, and cannot initiate connections
+to internal hosts that are outside the {product-title} cluster.
+- A Pod cannot reach specified internal subnets or hosts outside the {product-title} cluster that should be unreachable.
+
+An egress firewall policy is configured by creating an EgressNetworkPolicy Custom Resource (CR) object and specifying an IP address range in CIDR format or by specifying a DNS name.
+For example, you can allow one project access to a specified IP range but deny the same access to a different project, or you can restrict application developers from updating from Python pip mirrors, and force updates to come only from approved sources.
+
+[IMPORTANT]
+====
+You must have OpenShift SDN configured to use either the network policy or multitenant modes to configure egress firewall policy.
+
+If you are using network policy mode, egress policy is compatible with only one policy per project, and will not work with projects that share a network, such as global projects.
+====
+
+[CAUTION]
+====
+Exposing services by creating routes will ignore egress network policies.
+Egress network policy service endpoint filtering is done at the node `kube-proxy`.
+When the router is involved, `kube-proxy` is bypassed and egress network policy enforcement is not applied.
+====
+
+[id="limitations-of-egress-firewall"]
+== Limitations of egress firewall
+
+An egress firewall has the following limitations:
+
+* No project may have more than one EgressNetworkPolicy object.
```

Comment:
Tested both multitenet and network-policy mode in 4.3.0-rc.3, more than one egress network policies can be created per project.

Reply:
That's unexpected behavior... Can you please provide a yaml with the objects?

Reply:
@juanluisvaladas  Test log including policy yaml files: http://pastebin.test.redhat.com/831123

Reply:
Definitely it's not the correct behavior, I think the documentation should be merged anyway...

Reply:
Clarified above: we don't enforce this at object creation time, so you can create multiple ENPs, but it won't do what you wanted it to (and will cause the sdn pods to spew errors)